samples/videoserver/VideoServer2.py

```diff
@@ -0,0 +1,138 @@
+##+--------------------------------------------------------------------------
```

Comment:
A new, cleaned up, working version of the videoserver.py file but written as if people were watching.

Reply:
Neat!